.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+#### What does this PR do?
+#### Any background context you want to provide?
```

Comment:
Since big PR templates can be scary/demotivating, I'd combine this with first:
```
#### What does this PR do? (include background context, if relevant)
```